web/src/scenes/signup/signup.tsx

```diff
@@ -66,8 +68,27 @@ export function Signup() {
 
   const submit = useCallback(
     async (values: SignupFormValues) => {
-      console.log(values);
       const signup_token = localStorage.getItem("signup_token");
+      const ironcladId = crypto.randomUUID();
```

Comment:
@paolodamico Is it ok to have random UUID as signer id in Ironclad? 
We storing this value in Hasura.

Reply:
@paolodamico Also wanted to ask you if we have some ironclad staging secrets? to test everything